Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileCleaner
+    {
+        private const string _corruptionChar = "_";
+        private const string _emptyChar = "";
+        private readonly string _dirtyFile;
+        private readonly string _cleanedFile;
+        private readonly CultureInfo _cultureInfo;
+
+        public FileCleaner(string dirtyFile, string cleanedFile, CultureInfo cultureInfo)
+        {
+            _dirtyFile = dirtyFile;
+            _cleanedFile = cleanedFile;
+            _cultureInfo = cultureInfo;
+        }
+
+        public void Clean()
+        {
+            if (!Test.IsStringValid(_dirtyFile) || !Test.IsStringValid(_cleanedFile))
+            {
+                throw new ArgumentException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a constructor overloads that allows a more meaningful exception message to be provided.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)